usr/src/App.js

```diff
@@ -1,23 +1,120 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
 import './App.css';
-import logo from './logo.svg';
+import './index.css';
+
+const rootElement = document.getElementById('root');
+
+ReactDOM.render(<App />, rootElement);
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React DFDFDD
-        </a>
-      </header>
+    
```

Comment:
Consider to make a page component for a good practice so it looks like <div><Nav bar><body><footer></div>.